test/functional/apps/discover/group2/_data_view_edit.ts

```diff
@@ -78,10 +82,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect((await PageObjects.discover.getAllFieldNames()).length).to.be(3);
     });
 
-    it('update data view', async function () {
+    it('create saved data view', async function () {
```

Comment:
It turns out this was actually creating a new data view since `editDataView` used to call `clickCreateNewDataView` instead of `clickEditDataView`. I've updated `editDataView` to call `clickEditDataView` now, and changed this test to be a `create saved data view` test instead. The two new tests below will check for edits, ensuring things like the time range, hit counter, chart, and time filter are updated as expected.